src/Foundatio.Storage.SshNet/Storage/SshNetFileStorage.cs

```diff
@@ -211,18 +211,22 @@ public SshNetFileStorage(Builder<SshNetFileStorageOptionsBuilder, SshNetFileStor
 
             string[] userParts = uri.UserInfo.Split(new [] { ':' }, StringSplitOptions.RemoveEmptyEntries);
             string username = userParts.First();
```

Comment:
Can we just unescape the username and password by default `string username = Uri.UnescapeDataString(userParts.First())`